app/Jobs/DeleteQueryserviceNamespaceJob.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace App\Jobs;
+
+use App\QueryserviceNamespace;
+use App\Http\Curl\CurlRequest;
+use App\Http\Curl\HttpRequest;
+
+class DeleteQueryserviceNamespaceJob extends Job
+{
+    private $namespaceId;
+    private $request;
+
+    /**
+     * @return void
+     */
+    public function __construct($namespaceId = null, HttpRequest $request = null)
+    {
+        $this->request = $request ?? new CurlRequest();
+        $this->namespaceId = $namespaceId;
+    }
+
+    /**
+     * @return void
+     */
+    public function handle()
+    {
+        $qsNamespace = QueryserviceNamespace::whereId($this->namespaceId)->first();
+
+        if( !$qsNamespace ) {
+            $this->fail( new \RuntimeException("Namespace for {$this->namespaceId} not found.") );
+            return;
+        }
+
+        $queryServiceHost = config('app.queryservice_host');
+
+        $this->request->setOptions([
+            CURLOPT_URL => $queryServiceHost.'/bigdata/namespace/' . $qsNamespace->namespace ,
+            CURLOPT_RETURNTRANSFER => true,
+            CURLOPT_ENCODING => '',
+            CURLOPT_TIMEOUT => 10,
+            CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
+            // User agent is needed by the query service...
+            CURLOPT_USERAGENT => 'WBStack ProvisionQueryserviceNamespaceJob',
```

Comment:
this is wrong probably

Reply:
changed to DeleteQueryserviceNamespaceJob, not sure if there is any checks for this?

Reply:
There are not checks for this, but the UAs do show up in logs, which makes a useful name good!